optimism/src/mips/interpreter.rs

```diff
@@ -534,9 +520,37 @@ pub fn interpret_jtype<Env: InterpreterEnv>(env: &mut Env, instr: JTypeInstructi
 }
 
 pub fn interpret_itype<Env: InterpreterEnv>(env: &mut Env, instr: ITypeInstruction) {
-    let rs = env.get_instruction_part(InstructionPart::RS);
-    let rt = env.get_instruction_part(InstructionPart::RT);
-    let immediate = env.get_immediate();
+    let instruction = {
+        let instruction_pointer = env.get_instruction_pointer();
+        let v0 = env.read_memory(&instruction_pointer);
+        let v1 = env.read_memory(&(instruction_pointer.clone() + Env::constant(1)));
+        let v2 = env.read_memory(&(instruction_pointer.clone() + Env::constant(2)));
+        let v3 = env.read_memory(&(instruction_pointer + Env::constant(3)));
+        (v0 * Env::constant(1 << 24))
```

Comment:
iirc, other parts of the code (perhaps unmerged yet) used v4 as the most significant chunk instead